samples/jquery/00 ShoppingForm/webpack.config.js

```diff
@@ -0,0 +1,92 @@
+var path = require("path");
+var webpack = require("webpack");
+var HtmlWebpackPlugin = require('html-webpack-plugin');
+var ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+var basePath = __dirname;
+
+module.exports = {
+  context: path.join(basePath, "src"),
+  resolve: {
```

Comment:
If you want you can remove `resolve` section because [webpack](https://webpack.js.org/configuration/resolve/#resolve-extensions) has as default extensions `.js and .json`